### DISCUSSION TITLE
Initial State for Swipeable

### DISCUSSION THREAD

**Discussion by Kerockx** [software-mansion/react-native-gesture-handler#1243] [Q&A]
## Description

Hi,
Is it possible to set the initial state of a swipeable? I would like to have a specific swipeable row in a flatlist opened after rendering on the right.

For example:

row 1 -> closed
row 2 -> closed
row 3 -> openRight
row 4 -> closed

## Code

```js
const renderRightActions = (progress, dragX) => (
      <Icon style={styles.listIcon} ios='ios-checkmark-circle' android="md-checkmark-circle"/>     
  );

  const renderListItem = ({ item, index, rowMap }) => (
    <Swipeable
      renderRightActions={renderRightActions}
      <View style={styles.listRow}>
        <Text>{item.name}</Text>
      </View>
    </Swipeable>
  );

  return (
    <FlatList
    style={styles.flatList}
    data={categories}
    renderItem={renderListItem}
    keyExtractor={item => `${item.name}`}
    ItemSeparatorComponent={() => <View style={styles.listSeperator} />}
  />
  );
```
### Package versions

- React: 16.9.0
- React Native: 36.0.1
- React Native Gesture Handler: ~1.5.0

Thank you for your help!

Kerockx

**Comment by Jacek-gh** [comment]
I also would like to see left/right initial state properties added to Swipeable.  It is surprising for me that it is not yet there.
